src/styles/_card.scss

```diff
@@ -0,0 +1,151 @@
+.main__catalog {
+  max-width: 944px;
+  margin: 0 auto;
+  display: grid;
+  gap: 46px 48px;
+  justify-items: center;
+  grid-template-columns: 1fr;
+  margin-bottom: 72px;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 1fr);
+  }
+}
+
+.card {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  background-color: $four-color;
+  border-radius: 5px;
+  transition:
+    transform 300ms ease,
+    background-color 300ms ease;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__title {
+    color: $triple-color;
```

Comment:
Ensure that the variable `$triple-color` is defined in your SCSS files. If it's not, this will cause a compilation error.